Orange/widgets/data/oweditdomain.py

```diff
@@ -402,6 +406,10 @@ def __init__(self):
 
         box.layout().addWidget(self.editor_stack)
 
+        self.Warning.add_message(
```

Comment:
I think two variables with the same name should produce an error and require a fix before proceeding (instead of displaying a warning and working on)